swap/Selenium.txt

```diff
@@ -0,0 +1,51 @@
+div#pocks � ���� ���, � ������� �������� ����� pocks
```

Comment:
Maybe it'd be better to create file during test run instead of taking it from repository. But in this case some difficulties and unexpexted exceptions may occure, which should be handled.